libs/coin-modules/coin-cosmos/src/types/signer.ts

```diff
@@ -0,0 +1,34 @@
+export type CosmosAddress = {
+  publicKey: string;
+  address: string;
+};
+
+export type CosmosGetAddressAndPubKeyRes = {
+  bech32_address: string;
+  compressed_pk: string;
+  return_code: number; // TODO: validate type
+  error_message: string;
+};
+
+export type CosmosSignature = {
+  signature: null | Buffer;
+  return_code: number | string;
+};
+
+export type CosmosSignatureSdk = {
+  signature: Uint8Array;
+  return_code: number | string;
+};
+
+export interface CosmosSigner {
+  // getAddressAndPubKey(path: string, hrp: string, boolDisplay?: boolean): Promise<CosmosAddress>;
```

Comment:
Can't we remove the commented code?

Reply:
Done here https://github.com/LedgerHQ/ledger-live/pull/7636/commits/6b640d6c02629ba25a1f174a9773b12e7708bc13 🧹